packages/cli/src/commands/remove.ts

```diff
@@ -171,7 +174,7 @@ export default async function main(client) {
 
       projectDeployments
         .slice(0, 201)
-        .map(pDeployments => deployments.push(...pDeployments));
+        .map((pDeployments: any) => deployments.push(...pDeployments));
```

Comment:
Let's add the correct type to `projectDeployments` so you don't need any

Reply:
Fix 👍🏼